src/sidebar/util/query-string.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Generate a query string from a record of query parameters.
+ *
+ * The returned string does not have a leading "?" and the parameters are
+ * sorted by name.
+ *
+ * @example
+ *   stringify({ foo: 'bar', meep: 'one two' }) // Returns "foo=bar&meep=one+two"
+ *
+ * @param {Record<string, string>} params
```

Comment:
Why not allow number too?
```suggestion
 * @param {Record<string, string|number>} params
```

Reply:
URL parameters are always strings, so by forcing the caller to use strings here, it avoids any ambiguity.

Reply:
I debated allowing `number` here and relying on the implicit cast. Could have gone either way on this. Ended up deciding against it for now.